docs/crio.8.md

```diff
@@ -337,7 +337,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--seccomp-use-default-when-empty**: Use the default seccomp profile when an empty one is specified
 
-**--selinux**: Enable selinux support (default: false)
+**--selinux**: Enable selinux support (default: true)
```

Comment:
@jrvaldes this line is the problem

Reply:
I may need to read more about this. Are you suggesting that this change should be reverted (not included in the PR)?

Reply:
yeah. basically, the docs-validation make target runs against the config that's generated on your machine. If selinux is enabled, then this will be switched. you can run
```
alias replace_selinux="sed -i 's/Enable selinux support (default: true)/Enable selinux support (default: false)/g' completions/fish/crio.fish docs/crio.8.md"
```
which is what I do. This is part of the context for wanting a fix to https://github.com/cri-o/cri-o/issues/6499

Reply:
Understood. Addressed in 541502bc5860c170027b4124c297bf68f20c0988.